files/ansible-unbound-checkconf.sh

```diff
@@ -26,6 +26,7 @@ config_file_to_validate="${1}"
 version=`unbound-checkconf -h | grep -e '^Version' | cut -f 2 -d ' ' | cut -f 1,2 -d '.'`
 
 case "${version}" in
+    1.6)    option="-f -o directory" ;;
```

Comment:
to avoid listing known versions and options, try '-f' first, try '-o' if failed and exit if both failed.